frontend/src/Components/Hamburger.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react';
+import styles from './Hamburger.module.scss';
+import classNames from 'classnames';
+
+type HamburgerProps = {
+    open: boolean;
+};
+
+export const Hamburger = (props: HamburgerProps) => {
+    const [transformUpperRect, setTransformUpperRect] = useState('');
+    const [transformLowerRect, setTransformLowerRect] = useState('');
+    const dropRotate = () => {
+        setTransformUpperRect('');
+        setTransformLowerRect('');
+    };
+
+    useEffect(() => {
+        if (props.open) {
+          dropRotate();
+        } else {
+          setTransformUpperRect('rotate(45) translate(-4 -20)');
+          setTransformLowerRect('rotate(-45) translate(2 2)');
+        }  
```

Comment:
Extra trailing spaces:
```suggestion
        }
```